include/options.h

```diff
@@ -76,12 +76,13 @@ class Options {
     // Searches a list of filenames and replaces directories in the list with
     // all files descending from those directories. Files with extensions not in
     // the valid extensions list are excluded.
-    std::vector<fs::path> ExpandDirectories(const std::vector<fs::path>& filenames);
+    std::vector<fs::path> ExpandDirectories(const std::vector<fs::path>& filenames) const;
 
     // Filters out files listed in the --exclude command line switch. File paths
     // in the switch are evaluated relative to the current working directory
-    std::vector<fs::path> FilterExcludedFiles(std::vector<fs::path> filenames,
-                                              const std::vector<fs::path>& excludes);
+    static std::vector<fs::path> FilterExcludedFiles(
+		std::vector<fs::path> filenames,
```

Comment:
```suggestion
        std::vector<fs::path> filenames,
```
Tab indentations should be removed.